src/backend/backend.did

```diff
@@ -8,6 +8,11 @@ type ApiStringResult = variant {
     Err : ApiError;
 };
 
+type ApiNullResult = variant {
+    Ok : null;
+    Err : ApiError;
+};
+
```

Comment:
there's an `EmptyResult` already, so let's remove that and rename this to `ApiEmptyResult` (removing the `: null`)